spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  before(:each) do
+    @user =
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you fix other tests to pass even if they are not part of this milestone such as some `controllers tests`,...
![Screenshot from 2023-01-31 13-08-54](https://user-images.githubusercontent.com/83042742/215744303-4ffeb1d7-8f09-4f26-94f9-e18c8a7d16c8.png)